tensorflow_addons/image/tests/cutout_ops_test.py

```diff
@@ -33,7 +33,7 @@ def test_different_dtypes(dtype):
 
 
 def test_different_channels():
-    for channel in [0, 1, 3, 4]:
+    for channel in [1, 2, 3, 4]:
```

Comment:
It makes no sense to me to test 0 channel images. @fsx950223 Is this intended? Thanks.

Reply:
It's a edge test case.

Reply:
sure, let me checkout it.